src/components/core/PivotTable.tsx

```diff
@@ -154,15 +161,92 @@ export const getTreeLeaves = (tree: any, getChildren = (node: any) => node && no
     return leaves;
 };
 
+export const getSortItemByColId = (
+    execution: Execution.IExecutionResponses,
+    colId: string,
+    direction: 'asc' | 'desc'
+) => {
+    const dimensions: Execution.IResultDimension[] = execution.executionResponse.dimensions;
+    // sorting on column attribute measures will return colId like these 'a_2009_4-a_2071_12-m_3', 'a_2009'
+    // we need the last field 'm_3' or 'a_2009'
+    const fields = colId.split(FIELD_SEPARATOR);
```

Comment:
move these 5 lines to separate function and add some tests with typical inputs instead of comment

Reply:
done